### DISCUSSION TITLE
Date/Time field: date value does not display in datepicker in Safari

### DISCUSSION THREAD

**Discussion by twopiers** [statamic/cms#5895] [Troubleshooting]
I have an Events collection and a start_date_time field within it. Here is the YAML for that field in the events blueprint:
<img width="282" alt="Screen Shot 2022-04-22 at 11 28 30 AM" src="https://user-images.githubusercontent.com/1696177/164746062-34c0abca-d97a-45d9-8348-47afefbd785d.png">

Works great. However, when I edit an existing event, only the time shows up in the time selector. The date does not. This is only happening in Safari (currently I'm using 15.4)
<img width="292" alt="Screen Shot 2022-04-22 at 11 29 57 AM" src="https://user-images.githubusercontent.com/1696177/164746326-9f28cc0c-b9d0-4ff0-9ade-a17da3820a02.png">

If I save the event without interacting with the date calendar, the previously selected event date doesn't change (which is good), and I also don't get any kind of validation error when saving, even though this is marked as a required field.

**Comment by edalzell** [comment]
What version of Statamic?

**Reply by twopiers** [reply]
3.3.9

**Reply by edalzell** [reply]
Sounds like a bug, can you make an issue please?
<img width="290" alt="CleanShot 2022-04-30 at 11 56 05@2x" src="https://user-images.githubusercontent.com/6069653/166118992-350bcf00-7f66-4885-93f6-1b1599b25a65.png">

**Reply by edalzell** [reply]
I  just ran into this as well, issue [here](https://github.com/statamic/cms/issues/5976)
